### PR TITLE
QC for rice demographic model 

### PR DESCRIPTION
**Pull request by minesrebollo**
QC for rice demographic model (BottleneckMigration_3C07) #1459 #1453

### CODE REVIEW

File: stdpopsim/qc/OrySat.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+_species = stdpopsim.get_species("OrySat")
+
+def _BottMig_3C07_qc():
+    id = "QC-BottMig_3C07"
+    
+    #populations
+    rufipogon = stdpopsim.Population(id = "RUF")
+    indica = stdpopsim.Population(id = "IND")
+    japonica = stdpopsim.Population(id = "TRJ")
```
Comment (nspope) [inline stdpopsim/qc/OrySat.py:12]: These population objects need a description string, like for example:
https://github.com/popsim-consortium/stdpopsim/blob/0dcd497f8b78fcc73ede06691a53946ec6c7fd8b/stdpopsim/catalog/DroMel/demographic_models.py#L12-L18

without this, they'll error out

**Comment by minesrebollo** [comment]
@nspope, all checks have passed now!

**Comment by petrelharp** [comment]
It looks like we can merge this - @nspope or @minesrebollo please let me know if I've done something wrong!

**Comment by nspope** [comment]
Oh darn, thought I'd done that already-- thanks @petrelharp.

**Pull request merged**